tests/Select.test.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Select } from "../lib/components";
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+const options = [
+  {
+    label: "Option 1",
+    value: "option-1",
+  },
+  {
+    label: "Option 2",
+    value: "option-2",
+  },
+];
+
+describe("Select", () => {
+  it("should render without error", () => {
+    const { getByText } = render(<Select label="Select" options={options} />);
+    expect(getByText("Select")).toBeInTheDocument();
+  });
+
+  it("should show option list on clicking", () => {
+    const { container, getByText } = render(
+      <Select label="Select" options={options} />
+    );
+    userEvent.click(container.querySelector("input"));
+    expect(getByText("Option 1")).toBeInTheDocument();
+    expect(getByText("Option 2")).toBeInTheDocument();
+  });
+
+  it("should call onChange on select option", () => {
+    const onChange = jest.fn();
+    const { container, getByText } = render(
+      <Select label="Select" options={options} onChange={onChange} />
+    );
+    userEvent.click(container.querySelector("input"));
```

Comment:
Prefer using `getByRole` or `getByLabelText` instead of `container.querySelector`